src/EditorFeatures/CSharpTest/Diagnostics/ConditionalExpressionInStringInterpolation/CSharpAddParenthesesAroundConditionalExpressionInInterpolatedStringCodeFixProviderTests.cs

```diff
@@ -88,71 +88,71 @@ public async Task TestAddParenthesesWithTrivia()
         {
             await TestInMethodAsync(
                 @"var s = $""{ /* Leading1 */ true /* Leading2 */ ? /* TruePart1 */ 1 /* TruePart2 */[|:|] /* FalsePart1 */ 2 /* FalsePart2 */ }"";",
-                @"var s = $""{ /* Leading1 */ (true /* Leading2 */ ? /* TruePart1 */ 1 /* TruePart2 */: /* FalsePart1 */ 2 /* FalsePart2 */ )}"";");
+                @"var s = $""{ /* Leading1 */ (true /* Leading2 */ ? /* TruePart1 */ 1 /* TruePart2 */: /* FalsePart1 */ 2) /* FalsePart2 */ }"";");
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParenthesesAroundConditionalExpressionInInterpolatedString)]
         public async Task TestAddParenthesesClosingBracketInFalseCondition()
         {
             await TestInMethodAsync(
                 @"var s = $""{ true ? new int[0] [|:|] new int[] {} }"";",
-                @"var s = $""{ (true ? new int[0] : new int[] {} )}"";");
+                @"var s = $""{ (true ? new int[0] : new int[] {}) }"";");
```

Comment:
these results are actually preferred as we do not have the bogus space after the WhenFalse part.